### ISSUE TITLE
preserve json-ld scripts by default in rollup config

### ISSUE THREAD

**Issue by m4rrc0** [sw-yx/ssg#17]
When trying to add `json-ld` scripts to `svelte:head`, we get this issue:

```
[svelte-preprocess] Error transforming 'ld+json'.
```

I have documented my solution in [this stackoverflow](https://stackoverflow.com/a/59809388/6503854). It is ugly and I would be surprised there is not a better one. If anyone has a better solution it is welcome. In the meantime I suggest at least adding

```javascript
// rollup.config.js
// ...
import sveltePreprocess from 'svelte-preprocess';

const preprocess = sveltePreprocess({
  preserve: ['ld+json'],
  // ...
});

export default {
  client: {
    plugins: [
      svelte({
        preprocess,
        // ...
      }),
  },
  server: {
    plugins: [
      svelte({
        preprocess,
        // ...
      }),
    ],
  },
};
```
to the default rollup config.

Thoughts?

**Comment by swyxio** [comment]
sure thing. happy to take a PR or let me get to it when i'm cleaning up issues

**Comment by m4rrc0** [comment]
Happy to make a PR. #20 
Such a small change I did it directly from Github's UI. I hope I didn't forget anything obvious. ^^'

**Comment by m4rrc0** [comment]
Complementary PR on Sapper fork: https://github.com/sw-yx/sapper/pull/11

I'll let you close the issue @sw-yx in case there are more changes needed that I am not aware of.

**Comment by swyxio** [comment]
published as sapper fork 18, and included in ssg@0.0.61
